streamer/src/nonblocking/quic.rs

```diff
@@ -369,14 +371,21 @@ struct ConnectionEntry {
     exit: Arc<AtomicBool>,
     last_update: Arc<AtomicU64>,
     port: u16,
+    connection: Option<Connection>,
```

Comment:
@pgarg66 What is the reason that this is on `Option`?
Looks like out of tests we never use `None` here.

Reply:
Yeah, it was mostly for unit testing. It was not trivial to create a `Connection` object in the tests.